app/build.gradle

```diff
@@ -230,7 +237,10 @@ dependencies {
     //Test
     testCompile 'junit:junit:4.12'
     androidTestCompile 'org.assertj:assertj-core:1.7.1'
-    androidTestCompile ('com.android.support.test.espresso:espresso-core:2.2') {
+    androidTestCompile ('com.android.support.test.espresso:espresso-core:2.2.1') {
+        exclude module: 'support-annotations'
```

Comment:
You can also put conflict resolution above and we can remove these 2 excludes. I did it for Novoda droidcon London booth project. What do you think?